packages/stream-tests/src/__tests__/model.test.ts

```diff
@@ -20,9 +33,16 @@ describe('Model API http-client tests', () => {
   let ceramic: CeramicClient
 
   beforeAll(async () => {
+    ipfs = await createIPFS()
+  })
+
+  afterAll(async () => {
+    await ipfs.stop()
+  })
+
+  beforeEach(async () => {
```

Comment:
Does it really need to create a new Ceramic instance for each test, can't the test suite reuse the same instance?

Reply:
uh, I guess it can.  I changed it b/c Model creation is deterministic, so some of the tests are creating the same model and not actually really creating a new model, but just re-using the model created in an earlier test.  But I guess that doesn't actually really break any of the assumptions or invalidate anything that the tests are trying to test, at least not with the current set of tests, so I can put it back for now